File: python/pcats_api_example_manuscript_1.py
import pcats_api_client as pcats_api
 
#example 1

jobid=pcats_api.staticgp(datafile="../data/example1.csv", 
        outcome="Y",
        treatment="A",
        x_explanatory="X",
        x_confounding="X",
        burn_num=500,
        mcmc_num=500,
        outcome_type="Continuous",
        method="GP",
        tr_type="Discrete",
        pr_values="0,1,2")

print("JobID: {}".format(jobid))

status=pcats_api.wait_for_result(jobid)

if status=="Done":
    print(pcats_api.printgp(jobid))
else:
    print("Error")

File: R/pcatsAPIclientR/pcats_api_example_manuscript_1.R
library(pcatsAPIclientR)

#example 1

jobid <- pcatsAPIclientR::staticGP(datafile="../../data/example1.csv",
                                   outcome="Y",
                                   treatment="A",
                                   time="Time",
                                   x.explanatory="X",
                                   x.confounding="X",
                                   burn.num=500, mcmc.num=500,
                                   outcome.type="Continuous",
                                   method="GP",
                                   tr.type="Discrete",
                                   c.margin="0")

cat(paste0("JobID: ",jobid,"\n"))

status <- pcatsAPIclientR::wait_for_result(jobid)

if (status=="Done") {
   cat(pcatsAPIclientR::printgp(jobid))
}
